tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -8180,6 +8204,12 @@ node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.j
 node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.js(560,42): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.js(561,45): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.js(564,27): error TS2555: Expected at least 2 arguments, but got 1.
+node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.js(603,43): error TS2417: Class static side 'typeof HeapSnapshotRetainmentDataGrid' incorrectly extends base class static side 'typeof HeapSnapshotContainmentDataGrid'.
+  Types of property 'Events' are incompatible.
+    Type '{ ExpandRetainersComplete: symbol; }' is missing the following properties from type '{ ContentShown: symbol; SortingComplete: symbol; }': ContentShown, SortingComplete
+node_modules/chrome-devtools-frontend/front_end/profiler/HeapSnapshotDataGrids.js(603,43): error TS2417: Class static side 'typeof HeapSnapshotRetainmentDataGrid' incorrectly extends base class static side 'typeof HeapSnapshotContainmentDataGrid'.
+  Types of property 'Events' are incompatible.
+    Type '{ ExpandRetainersComplete: symbol; }' is not assignable to type '{ ContentShown: symbol; SortingComplete: symbol; }'.
```

Comment:
Static side inheritance errors brought on by better usage of `getMergeSymbol` in the js declaration emitter PR. We _should_ have always been issuing these.